.github/workflows/test.yml

```diff
@@ -11,6 +11,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
```

Comment:
Given that the Go module we're installing is what defines the required Go version, this should be part of the composite action and not left to the user

Reply:
Moved to the action